src/modules/job-info/job_state.c

```diff
@@ -1673,6 +1760,111 @@ int job_state_init_from_kvs (struct info_ctx *ctx)
     return 0;
 }
 
+static void job_events_priority (struct job_state_ctx *jsctx,
+                                 flux_jobid_t id,
+                                 double timestamp,
+                                 json_t *context)
+{
+    struct job *job;
+    int priority;
+
+    if (!context) {
+        flux_log_error (jsctx->h, "%s: no priority context for priority event",
+                        __FUNCTION__);
+        return;
+    }
+
+    if (json_unpack (context, "{ s:i }", "priority", &priority) < 0) {
+        flux_log_error (jsctx->h, "%s: priority event context invalid",
```

Comment:
errno not set by `json_unpack()`, don't call `flux_log_error()`

Reply:
Missed that one.

Reply:
ugh, i went ahead and did a big audit of the whole file.  A bunch were due to cut & pastes from other areas.  I'll add as a separate commit.